www/templates/iframe/MediaPlayer.tpl.php

```diff
@@ -266,8 +266,16 @@ function canAccessParent() {
                             var value = $(this).find(":selected").val();
                             t.setTrack(value);
                         });
+                    };
 
+                    function fixTheControls(){
+                        if (t.media.duration != NaN) {
+                            t.setControlsSize();
+                            m.removeEventListener('timeupdate', fixTheControls);
```

Comment:
This isn't working as expected for me. I'm pretty sure it is because of a race condition.

On the video I'm testing, this triggers before time is updated in the UI, thus still resulting the same issue.